server/app/assets/javascripts/phone.ts

```diff
@@ -19,11 +19,13 @@ function formatPhone(element: HTMLElement) {
 }
 
 export function init() {
-  const element = document.getElementById('cf-phone-number')
-  const countryElement = document.getElementById('cf-phone-country-code')
-  if (element && countryElement) {
-    formatPhone(element)
-  }
+  document.querySelectorAll('[id^=cf-phone-number]').forEach((el) => {
+    // Our questions automatically include an element matching id appended with "-errors"; filter them out.
```

Comment:
praise: helpful comment! And good to know.

Reply:
suggestion (non-blocking): add a sentence to this comment explaining that the phone elements will have a unique id with a number prepended on the end which is why you have to select them this way and then filter out the `error` ones.